src/tests/integration/accessToken.test.ts

```diff
@@ -0,0 +1,39 @@
+import { startAppServer } from '../../server/server'
+import uuidv4 from 'uuid/v4'
+import { accessTokensAdapter } from '../helpers/accessTokensAdapter'
+
+const TEST_PORT = process.env.PORT || 50605
+const ADMIN = process.env.ADMIN || 'hjhj'
+
+describe('/api/accessTokens', () => {
+  let server
+
+  beforeAll(async () => {
+    server = await startAppServer(TEST_PORT, ADMIN)
+  })
+
+  afterAll(() => {
+    server.close()
+  })
+
+  it('checks to see if accesstoken is created', async () => {
+    const name = `c0d3access${uuidv4()}`
+    const postResponse = await accessTokensAdapter('/', 'POST', {
+      name
+    })
+    const postAccessToken = await postResponse.json()
+    expect(postAccessToken.name).toEqual(name)
```

Comment:
just because we get back a response, does not mean 100% its saved. Could we send a `GET` request here to get all the access tokens, then make sure the one we just created is in the accessToken list?